pages/Zotero.xml

```diff
@@ -148,6 +155,10 @@ inserted in the fields: author, editor and translator.</p>
            etc.) are not needed unless their removal would mislead the
            reader.</p>
                     </item>
+                    <item>
+                     <p>In case if the transliterated form of a contributor's name in the publication differs from the transliterated form according to the rules applied by the project, enter the standardized form in accordance with http://viaf.org/ into the corresponding field.
+                     In addition to that, ALWAYS provide in notes the actual spelling of a contributor's name, as it appears in the publication.</p>
```

Comment:
please try to specify that the note is the one which a user can create in Zotero under Notes and not any other form of note